docs/options api.md

```diff
@@ -85,6 +85,20 @@ class ExampleWorld(World):
     options: ExampleGameOptions
 ```
 
+### Option Groups
+Options may be categorized into groups for display on the WebHost. The name of a group may be defined by adding a
+`group_name` to your option. Option groups are displayed alphabetically on the player-options and weighted-options
+pages. Options without a group name are categorized into a generic "Game Options" group.
```

Comment:
Is this outdated information?
I tried doing this and it didn't seem to do anything.

I instead defined the list on my `WebWorld` and it did work.

(I like the `WebWorld` API better anyway.)

Reply:
Oh yeah, that was the old bad implementation. I thought @alwaysintreble changed the documentation as well when implementing the new, better version. I guess it slipped through the cracks.

Reply:
I have updated the documentation.

Reply:
I have updated the documentation.